pkg/operator/ceph/cluster/osd/osd_test.go

```diff
@@ -393,6 +393,10 @@ func TestGetOSDInfo(t *testing.T) {
 
 	d2, _ := c.makeDeployment(osdProp, osd2, dataPathMap)
 	osds2, _ := c.getOSDInfo(d2)
+	commands := c.getPauseContainerCommands([]string{"echo"}, []string{"test"})
```

Comment:
We need a test when `PAUSE_OSD` is true.

Reply:
sure.

Reply:
@leseb can you suggest me where to add the test corresponding to the `PAUSE_OSD = true`.